org.idempiere.webservices/WEB-INF/src/org/compiere/model/MWebServiceType.java

```diff
@@ -286,5 +286,45 @@ public X_WS_WebServiceFieldInput getFieldInput(String colName){
 		
 		return m_inputFieldMap.get(colName);
 	}
+
+	public static void insertParameters (Properties ctx, MWebServiceType ws, String trxName) {
```

Comment:
it's usually preferable to make this methods not static, so, you can simply call
ws.insertParameters() - no need to pass the ctx and trx as they are part of the object.

Reply:
Thanks for the tip